Library/Formula/relx.rb

```diff
@@ -0,0 +1,18 @@
+class Relx < Formula
+  desc "Sane, simple release creation for Erlang"
+  homepage "https://github.com/erlware/relx"
+  url "https://github.com/erlware/relx/archive/v3.5.0.tar.gz"
+  sha256 "60d7252369325c4a1146bc443383df2692afa885ffe10c6def20e5aefcb142ff"
+
+  depends_on "erlang"
+
+  def install
+    system "./rebar3", "update"
+    system "./rebar3", "escriptize"
+    bin.install "_build/default/bin/relx"
+  end
+
+  test do
+    system "#{bin}/relx", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. e.g. write out a `relx.config` and do something with it. Thanks1